widgets/post-carousel/js/carousel.js

```diff
@@ -23,7 +17,7 @@ jQuery( function ( $ ) {
 				arrows: false,
 				infinite: false,
 				rows: 0,
-				rtl: direction == 'rtl',
+				rtl: $$.attr( 'dir' ) == 'rtl',
```

Comment:
I missed this before, but shouldn't this be a data attribute? So `data-dir`?

Reply:
Resolved via https://github.com/siteorigin/so-widgets-bundle/commit/f0ae7eed63c7f5a17e2e19070f98a9cdaf6cfdf2.